### PR TITLE
Release 5.2.0

### PR DESCRIPTION
**Pull request by swapnil-verma-gl**
**JIRA ticket link or changeset's description**
Release ACA 5.2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,16 +12,17 @@ Please refer to the public [documentation](https://alfresco-content-app.netlify.
...
-| 5.0.x | 7.0.0-alpha.2 | 23.3      | 18.x | 15.x    |
-| 4.4.x | 6.7           | 23.2      | 18.x | 14.x    |
-| 4.3.x | 6.4           | 23.1      | 18.x | 14.x    |
-| 4.2.x | 6.3           | 23.1.0-M4 | 18.x | 14.x    |
-| 4.1.x | 6.2           | 7.4       | 18.x | 14.x    |
-| 4.0.x | 6.1           | 7.4       | 14.x | 14.x    |
-| 3.1.x | 5.1           | 7.3       |      |         |
-| 3.0.x | 5.0           | 7.3       |      |         |
+|-------|---------------|-----------|------|--------|
+| 5.2.x | 7.0.0-alpha.6 | 23.3      | 18.x | 16.x   |
```
Comment (MichalKinas) [inline README.md:16]: We released ACS 23.4 now so this will be compatible version

File: docs/README.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ The documentation is divided into the following sections:
 
 | ACA   | ADF           | ACS       | Node | Angular |
 |-------|---------------|-----------|------|---------|
+| 5.2.x | 7.0.0-alpha.6 | 23.3      | 18.x | 16.x    |
```
Comment (MichalKinas) [inline docs/README.md:43]: We released ACS 23.4 now so this will be compatible version

File: projects/aca-content/package.json [hunk 3]
```diff
@@ -5,10 +5,10 @@
   "peerDependencies": {
     "@angular/common": ">=15.2",
     "@angular/core": ">=15.2",
-    "@alfresco/adf-core": ">=7.0.0-alpha.5-0",
-    "@alfresco/adf-content-services": ">=7.0.0-alpha.5-0",
-    "@alfresco/adf-extensions": ">=7.0.0-alpha.5-0",
-    "@alfresco/js-api": ">=8.0.0-alpha.5-0",
+    "@alfresco/adf-core": ">=7.0.0-alpha.6-0",
```
Comment (MichalKinas) [inline projects/aca-content/package.json:8]: During the release those version should point to the release version directly so `>=7.0.0-alpha.6` then we will change to version to `>=7.0.0-alpha.7-0` in post release version bump PR

File: projects/aca-shared/package.json [hunk 4]
```diff
@@ -5,10 +5,10 @@
   "license": "LGPL-3.0",
   "scripts": {},
   "peerDependencies": {
-    "@alfresco/adf-content-services": ">=7.0.0-alpha.5-0",
-    "@alfresco/adf-core": ">=7.0.0-alpha.5-0",
-    "@alfresco/adf-extensions": ">=7.0.0-alpha.5-0",
-    "@alfresco/js-api": ">=8.0.0-alpha.5-0",
+    "@alfresco/adf-content-services": ">=7.0.0-alpha.6-0",
```
Comment (MichalKinas) [inline projects/aca-shared/package.json:8]: Same as above

File: README.md (hunk 1 above)
Comment (swapnil-verma-gl) [reply README.md]: Updated

File: docs/README.md (hunk 2 above)
Comment (swapnil-verma-gl) [reply docs/README.md]: Updated

File: projects/aca-shared/package.json (hunk 4 above)
Comment (swapnil-verma-gl) [reply projects/aca-shared/package.json]: Updated

File: projects/aca-content/package.json (hunk 3 above)
Comment (swapnil-verma-gl) [reply projects/aca-content/package.json]: Updated

**Review by nikita-web-ua — APPROVED** [review]

**Review by MichalKinas — APPROVED** [review]

**Pull request merged**